pyvelox/pyvelox.cpp

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "pyvelox.h" // @manual
+
+namespace facebook::pyvelox {
+using namespace velox;
+namespace py = pybind11;
+
+std::string serializeType(const std::shared_ptr<const velox::Type>& type) {
+  const auto& obj = type->serialize();
+  return folly::json::serialize(obj, velox::getSerializationOptions());
```

Comment:
where's the folly/json include?

Reply:
got the same question before. seems more internal related?

Reply:
Koski has this ; trying to keep velox/koski/ta to use similar code and thus the folly::json.

Reply:
back to my comment: shouldn't we have a `#include "folly/json.h"` here somewhere?